frontend/src/components/MainHeader/index.jsx

```diff
@@ -25,6 +26,45 @@ const SiteHeader = () => {
   const [searchText, setSearchText] = useState(null);
   const [userId, setUserId] = useState();
 
+  const notifications = [
```

Comment:
We should connect to the backend before merging, since the necessary endpoints are implemented since this push. I am requesting a review.